sdk/servicebus/Azure.Messaging.ServiceBus/src/Primitives/ServiceBusReceivedMessage.cs

```diff
@@ -203,7 +268,23 @@ public class ServiceBusReceivedMessage
         ///     the topmost 16 bits reflect the partition identifier. Sequence numbers monotonically increase.
```

Comment:
nit: This seems dangerously close to an implementation detail.  Are we really comfortable with this being part of our public contract?  Do we know that the service guarantees this to be a stable contract from their side?  _(I realize this is probably brought over from T1, but ....)_

Reply:
I think this is straight out of the service documentation.